R/survey_statistics.r

```diff
@@ -334,12 +334,20 @@ get_var_est <- function(stat, vartype, var_names = "", grps = "",
       se
     } else if (vvv == "ci") {
       if (!quantile) {
-        ci <- data.frame(matrix(confint(stat, level = level),
-                                ncol = 2 * out_width))
+        if (length(level)==1) {
+          ci <- data.frame(matrix(confint(stat, level = level),
```

Comment:
Also, can you change these two instances of `confint` to `stats::confint`? I made this change because of an R CMD check note, and now we've got a git merge conflict. If you change it, then I can just accept your version.